src/api/usage-consolidation.js

```diff
@@ -0,0 +1,48 @@
+import {
+  getHeaders,
+  doRequest,
+  createUrl,
+} from './common';
+
+const API_URL = '/eholdings/kb-credentials';
```

Comment:
I think we also need to add `/uc` at the end here

Reply:
End point `/eholdings/kb-credentials/uc` will be unvalid and wrong for our backend